tests/inspectdb/tests.py

```diff
@@ -381,9 +382,9 @@ def test_foreign_data_wrapper(self):
                     sepal_length real,
                     sepal_width real
                 ) SERVER inspectdb_server OPTIONS (
-                    filename '/dev/null'
+                    filename %s
                 )
-            ''')
+            ''', [os.devnull])
```

Comment:
If you split this small test fix into separate commits with a title like "Made inspectdb tests pass on PostgreSQL Windows", it could be merged to master ahead of the main PR. Same with some other fixes like `databases = '__all__'` on the checks tests and the `with tempfile...`

Reply:
Already done in #13102. This PR just needs rebasing.

Reply:
👍

Reply:
The ```databases='__all__'``` change isn't a fix. It's a needed change since we're querying the databases every time we do run system checks because of the change in the ```my_check``` test. 

Will do for the ```with tempfile```!